Dockerfile

```diff
@@ -85,14 +85,18 @@ COPY --from=golang:bullseye /usr/local/go /usr/local/go
 WORKDIR /go/src/app
 
 COPY go.mod go.sum ./
-RUN go get -v ./...
+RUN go mod graph | awk '{if ($1 !~ "@") print $2}' | xargs go get
 
 RUN mkdir caddy && cd caddy
-COPY go.mod go.sum ./
+COPY caddy/go.mod caddy/go.sum ./caddy/
 
-RUN go get -v ./...
+RUN cd caddy && go mod graph | awk '{if ($1 !~ "@") print $2}' | xargs go get
 
-COPY . .
+COPY *.* .
```

Comment:
I changed this to `*.*` because my editor likes to put things in a dotfile, which caused the cache to always blow up.